Static/Row.swift

```diff
@@ -11,47 +11,47 @@ public struct Row: Hashable, Equatable {
     /// Representation of a row accessory.
     public enum Accessory: Equatable {
         /// No accessory
-        case None
+        case none
 
         /// Chevron
-        case DisclosureIndicator
+        case disclosureIndicator
 
         /// Info button with chevron. Handles selection.
-        case DetailDisclosureButton(Selection)
+        case detailDisclosureButton(Selection)
 
         /// Checkmark
-        case Checkmark
+        case checkmark
 
         /// Info button. Handles selection.
-        case DetailButton(Selection)
+        case detailButton(Selection)
 
         /// Custom view
-        case View(UIView)
+        case view(UIView)
```

Comment:
I noticed the Swift 3 migrator missed this and I had to make the change by hand. So weird.